actors/abi/sector.go

```diff
@@ -92,9 +92,14 @@ type PartialTicket []byte // 32 bytes
 type PoStVerifyInfo struct {
 	Randomness      PoStRandomness
 	CommR           cid.Cid
-	Candidates      []PoStCandidate      // From OnChain*PoStVerifyInfo
-	Proofs          []PoStProof          // From OnChain*PoStVerifyInfo
-	EligibleSectors map[SectorID]cid.Cid // TODO: HAMT?
+	Candidates      []PoStCandidate // From OnChain*PoStVerifyInfo
+	Proof           []byte
+	EligibleSectors []SectorInfo
+}
+
+type SectorInfo struct {
+	SectorID SectorNumber
+	CommR    cid.Cid
```

Comment:
@whyrusleeping - Suggestion:

- change `CommR` field name to `SealedCID` (to match elsewhere in spec)
- change `SectorID` field name to `SectorNumber` (to match elsewhere in spec)